tests/test_cl_half.h.c

```diff
@@ -95,7 +95,7 @@ int main(void)
   CHECK_FROM_FLOAT(-(CL_HALF_MAX + 1000.f), 0xfbff, RTP);
   CHECK_FROM_FLOAT(-(CL_HALF_MAX + 1000.f), 0xfc00, RTN);
   CHECK_FROM_FLOAT(-(CL_HALF_MAX + 1000.f), 0xfbff, RTZ);
-#ifndef _MSC_VER
+#if 0 // Hexadecimal float constant is C++17
```

Comment:
This has been introduced in C99 and I checked that it is supported in C11

Reply:
Are you OK merging this as-is to unblock CI?  We can fix this and any other minor issues in a subsequent PR.

Reply:
I guess I would, but this removes tests that maybe are only done here, so we must not forget about it.